scripts/gzip-all.js

```diff
@@ -0,0 +1,30 @@
+const path = require('path');
+const gzipAll = require('gzip-all');
+
+var walkSync = function(dir, filelist) {
+    if (dir[dir.length - 1] != path.sep) dir = dir.concat(path.sep);
```

Comment:
What's with adding `path.sep`? This sounds redundant

Reply:
`path.join` handles adding separators AFAIK

Reply:
It does? Okay, I'll make changes.

Reply:
Done.